engines/dreamweb/talk.cpp

```diff
@@ -115,6 +122,11 @@ void DreamWebEngine::startTalk() {
 }
 
 const uint8 *DreamWebEngine::getPersonText(uint8 index, uint8 talkPos) {
+	Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
+	if (ttsMan != nullptr && ConfMan.getBool("tts_enabled") && !hasSpeech()) {
+        ttsMan->say((const char*)_personText.getString(index*64 + talkPos),
+		Common::TextToSpeechManager::INTERRUPT);
+    }
```

Comment:
The indentation seems off on a couple of lines here. You may be using spaces instead of tabs.

Also maybe it would be worth using a variable for the text.
```c++
const uint8 *text = (const uint8 *)_personText.getString(index*64 + talkPos);
Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
if (ttsMan != nullptr && ConfMan.getBool("tts_enabled") && !hasSpeech())
	ttsMan->say((const char*)text),	Common::TextToSpeechManager::INTERRUPT);
return text;

Reply:
Also I am wondering if it might be better to use the `TextToSpeechManager` in places where this function is called rather than here. And I see you already do that in `startTalk()`. Doesn't that cause some sentences to be spoken twice?